ecs/users/fields/custom/policy.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: policy
+  title: Wazuh's User Policy
+  description: Policy assigned to a role.
+  reusable:
+    top_level: true
```

Comment:
I don't think a policy would be added at top level.

Reply:
Fixed on commit https://github.com/wazuh/wazuh-indexer-plugins/pull/303/commits/46c50892690e8669afab8fe1552f2b409a4af745